### PR TITLE
Eazycard

### PR DESCRIPTION
**Pull request by BBriset**
Building header/ user avatar components/ Eazycard and profile pages

### CODE REVIEW

**Review by HeloiseFerla — DISMISSED** [review]
UserProfile or UserAvatar?

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+import UserProfile from 'components/UserProfile';
+import { Link } from 'react-router-dom';
+import SHeader from './style';
+import home from '../../assets/Img/home.svg';
+
+export default function Header({ logo }) {
+  return (
+    <SHeader>
+      <Link to="/home">
+        <img src={home} alt="ico" />
```
Comment (Hacene010) [inline src/components/Header/index.jsx]: What does ico mean ?

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+      </Link>
+      <img className="logo" src={logo} alt="logo" />
+      <Link to="/profilpage">
```
Comment (Hacene010) [inline src/components/Header/index.jsx]: Forgot the camelCase

File: src/components/UserProfile/index.jsx [hunk 3]
```diff
@@ -1,3 +1,10 @@
+import SUserProfile from './style';
+import user from '../../assets/Img/users/e3.png';
+
 export default function UserProfile() {
-  return <p>UserProfile</p>;
+  return (
+    <SUserProfile>
+      <img classeName="userImg" src={user} alt="user.name" />
```
Comment (Hacene010) [inline src/components/UserProfile/index.jsx]: camelCase for the alt plz

File: src/pages/EazyCard/index.jsx [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+import nfc from '../../assets/Img/iphone.svg';
+import logo from '../../assets/Img/easyApp.png';
+import card from '../../assets/Img/easyCard.png';
+
 export default function EazyCard() {
-  return <p>EazyCard</p>;
+  return (
+    <>
+      <Header logo={logo} />
+
```
Comment (Hacene010) [inline src/pages/EazyCard/index.jsx]: empty line !!!!?

File: src/pages/EazyCard/style.jsx [hunk 5]
```diff
@@ -1,5 +1,74 @@
...
+    
+  }
+  .nfc {
+    bottom: -12px;
+    position: relative;
+    width: 65px;
+    height: 70px;
+    animation: phoneNfc 5s infinite;
+    perspective: 1000px;
+
```
Comment (Hacene010) [inline src/pages/EazyCard/style.jsx]: empty line !!!!

File: src/pages/Profile/index.jsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+import SProfilPage from './style';
+import apple from '../../assets/Img/apple.png';
+import samsung from '../../assets/Img/samsung.png';
+import applePay from '../../assets/Img/Apple_Pay_logo.svg';
+import cb from '../../assets/Img/logo-cb.svg';
+import larrondi from '../../assets/Img/LARRONDI.svg';
+import logo from '../../assets/Img/easyApp.png';
+import card from '../../assets/Img/easyCard.png';
+
+const solde = '37,64€';
```
Comment (Hacene010) [inline src/pages/Profile/index.jsx:12]: Don't forget to replace this absolute value

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Seen some evils. about coherance in your style especially ( empty lines sometimes ).

File: src/pages/Profile/index.jsx (hunk 6 above)
Comment (BBriset) [reply src/pages/Profile/index.jsx:12]: yes the logic for that part isnt already implemented, that why it's static at the moment

**Review by HeloiseFerla — APPROVED** [review]
ok

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**